Gopkg.lock

```diff
@@ -213,6 +213,15 @@
   revision = "f611eb38b3875cc3bd991ca91c51d06446afa14c"
   version = "v1.3.0"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/lib/pq"
```

Comment:
I'd rather used https://github.com/jackc/pgx instead of `lib/pq`

Reply:
@whisper-bye thanks for your contribution, `upper/db` is too much for our use case, we need a tinier one as. In addition to that ideally if postgres being supported we don't need the polling implementation given PG has notify/listen mechanism, so the PG implementation would have this feature

Reply:
thanks, close this PR